packages/www/pages/dashboard/usage/index.tsx

```diff
@@ -47,7 +51,14 @@ const Usage = () => {
 
   const doSetTimeStep = async (ts: string) => {
     setTimestep(ts);
-    const [res, usage] = await getBillingUsage(from, to, null, ts);
+    const [
+      res,
+      usage = {
+        TotalUsageMins: 0,
+        DeliveryUsageMins: 0,
+        StorageUsageMins: 0,
+      },
```

Comment:
I think this default is redundant with the one you already added to `getBillingUsage` impl?